tests/test_updater_ng.py

```diff
@@ -109,14 +114,94 @@ def tearDown(self):
     # Logs stdout and stderr from the sever subprocess.
     self.server_process_handler.flush_log()
 
+  def _create_consistent_target(self, delegator_filename: str, targetname: str):
+    """Create consistent targets copies of their non-consistent counterparts
+    inside the repository directory.
+
+    Args:
+      delegator_filename: Name of the delegator metadata file.
+      targetname: Name of the target file.
+
+    """
+    delegator_path = os.path.join(self.client_directory, delegator_filename)
```

Comment:
You keep using this delegator to get the hashes, I keep disagreeing. We need a second opinion.

Reply:
I am sorry it seemed that I disagree. I am just not sure how can I get information which hashes needs to be calculated for
a particular delegation if I don't access the delegator.
It seems I am missing something.
Can you share more how can I simplify this without accessing the delegator?